pkgs/development/libraries/libint/default.nix

```diff
@@ -52,23 +155,44 @@ let
 
     buildInputs = [ boost eigen ];
 
-    preConfigure = "./autogen.sh";
-
-    configureFlags = [
-      "--enable-eri=2"
-      "--enable-eri3=2"
-      "--enable-eri2=2"
-      "--with-eri-max-am=7,5,4"
-      "--with-eri-opt-am=3"
-      "--with-eri3-max-am=7"
-      "--with-eri2-max-am=7"
-      "--with-g12-max-am=5"
-      "--with-g12-opt-am=3"
-      "--with-g12dkh-max-am=5"
-      "--with-g12dkh-opt-am=3"
-      "--enable-contracted-ints"
-      "--enable-shared"
-    ] ++ lib.optional enableFMA "--enable-fma";
+    preConfigure = with lib.strings; ''
+      ./autogen.sh
+
+
+      configureFlagsArray+=(
+        ${optionalString enableFMA "--enable-fma"}
```

Comment:
This should be a list.

Reply:
Correct. Had spaces at some point in the arguments. Is a list again.